components/breadcrumb/style/patch.less

```diff
@@ -20,8 +20,14 @@ nz-breadcrumb {
 
 nz-breadcrumb-item:last-child {
   color: @breadcrumb-last-item-color;
+  a {
+    color: @breadcrumb-last-item-color;
+    &:hover {
+      color: @breadcrumb-last-item-color;
+    }
+  }
```

Comment:
```suggestion

```
Just modify `& > @{breadcrumb-prefix-cls}-item:last-child` to `& > nz-breadcrumb-item:last-child` on Line 9.

Reply:
ok